internal/pkg/describe/service.go

```diff
@@ -318,23 +320,28 @@ func (d *appRunnerServiceDescriber) Service() (*apprunner.Service, error) {
 
 // IsPrivate returns true if the service is configured as non-public.
 func (d *appRunnerServiceDescriber) IsPrivate() (bool, error) {
-	vicArn, err := d.vpcIngressConnectionARN()
+	_, err := d.vpcIngressConnectionARN()
 	if err != nil {
+		if errors.Is(err, errVPCIngressConnectionNotFound) {
+			return false, nil
+		}
+
 		return false, err
 	}
 
-	return vicArn != nil, nil
+	return true, nil
 }
 
 // ServiceURL retrieves the app runner service URL.
 func (d *appRunnerServiceDescriber) ServiceURL() (string, error) {
-	vicArn, err := d.vpcIngressConnectionARN()
-	if err != nil {
+	vicARN, err := d.vpcIngressConnectionARN()
+	isErrNotFound := errors.Is(err, errVPCIngressConnectionNotFound)
```

Comment:
```suggestion
	isVICNotFound := errors.Is(err, errVPCIngressConnectionNotFound)
```
hopefully this makes it clearer in L343 that vicFound => private => private URL 🤔 ?